UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/RNWRepository.cs

```diff
@@ -83,23 +83,13 @@ public async Task<DateTime> GetRadioNavigationalWarningsLastModifiedDateTime()
             return DateTime.MinValue;
         }
 
-        public EditRadioNavigationalWarningAdmin GetRadioNavigationalWarningById(int id)
+        public RadioNavigationalWarning GetRadioNavigationalWarningById(int id)
         {
             RadioNavigationalWarning rnwWarning = _context.Set<RadioNavigationalWarning>().Find(id);
```

Comment:
Can directly return result and then rnwWarning  variable can be removed.
```suggestion
            return _context.Set<RadioNavigationalWarning>().Find(id);
```

Reply:
changes done